examples/demos/example13-all-features.js

```diff
@@ -125,15 +125,15 @@ export default class extends React.Component {
     };
   }
 
-  createRows = (numberOfRows) => {
```

Comment:
sorry for my eslint config, will disable it after code review, to set those back